src/dashboard/components/navbar_component.py

```diff
@@ -99,3 +99,14 @@ def navbar_component() -> Component:
 def update_navbar(path_name: str) -> list[Component]:
     """Update the selected navbar item based on the current url."""
     return generate_navbar_items(dash.page_registry, path_name)
+
+
+@callback(
+    Output("main-navbar", "className"), Input("url", "pathname"), State("main-navbar", "className")
+)
+def show_navbar(path_name: str, class_name: str) -> list[Component]:
```

Comment:
It might be a good idea to generalize this functionality. One way would be a page registry parameter that toggles whether the navbar should be visible on that page. `show_navbar` or similar. If this solution is used the callback could still be in `navbar_component.py`.